passiveData/build.gradle.kts

```diff
@@ -25,28 +51,64 @@ kotlin {
     sourceSets {
         val commonMain by getting {
             dependencies {
+                implementation(Deps.Coroutines.core) {
+                    version {
+                        strictly(Versions.kotlinCoroutines)
+                    }
+                }
+
                 implementation(Deps.KotlinX.dateTime)
-                api (Deps.KotlinX.serializationJson)
+                api(Deps.KotlinX.serializationJson)
+
+                implementation(Deps.Ktor.clientCore)
+                //Is api to give depending modules access to JsonElement
+                api(Deps.Ktor.clientSerialization)
+                implementation(Deps.Ktor.clientLogging)
+
+                // koin
+                api(Deps.Koin.core)
+
+                implementation(Deps.Napier.napier)
+
+                implementation("com.google.android.gms:play-services-location:18.0.0")
+
             }
         }
         val commonTest by getting {
             dependencies {
                 implementation(kotlin("test-common"))
                 implementation(kotlin("test-annotations-common"))
+
+                implementation(Deps.Ktor.clientMock)
+
+                implementation(Deps.Ktor.clientMock)
+                api(Deps.Koin.test)
             }
         }
         val androidMain by getting {
             dependencies {
+                api(Deps.Coroutines.android)
+
                 api(Deps.AssessmentModel.sdk)
+                implementation(Deps.Ktor.clientAndroid)
+                implementation(Deps.Koin.android)
+
+                // TODO: experimental
```

Comment:
sign and date all TODO's

Reply:
Removed this. TODO was meant to remind me to delete :)